cmd/tfengine/README.md

```diff
@@ -91,70 +92,88 @@ Use our [example](../../examples/tfengine) configs to quickly get started.
 The engine takes a path to an input config and a path to output the generated
 Terraform configs. For details on fields for the input schema, see the
 [schema](../../internal/tfengine/schema.go). After the output has been generated,
-there is no dependency on the engine any longer, and the user can directly use
+there is no longer a dependency on the engine and the user can directly use
 the `terraform` and `terragrunt` binaries to deploy the infrastructure.
 
-Replace the values in the [example](../../examples/tfengine/simple.hcl) with
-values for your infrastructure, then run the following commands:
+1. Replace the values in the [example](../../examples/tfengine/simple.hcl) with
```

Comment:
This should go after the next step because you get the example locally after cloning the repo

Reply:
Done - thanks!